tests-wtr/dom-actions/prehide/dom-actions.prehide.empty-selector.ts

```diff
@@ -0,0 +1,27 @@
+import { expect } from '@esm-bundle/chai';
+import { DomActions } from '../../../lib/dom-actions';
+import { instantiateDomActions } from '../utils';
+
+// must be run from dom-actions.prehide.empty-selector.html
+describe('prehide', () => {
+    let domActions: DomActions;
+
+    beforeEach(() => {
+        // Given
+        domActions = instantiateDomActions();
+    });
+
+    it('returns false when selector is empty string', () => {
```

Comment:
interesting, empty selector was never supposed to be passed here I think. In fact, right now it produces an invalid CSS in the style tag.
Invalid CSS is ignored by the browser, and we don't pass empty selectors anyways, so it's not a big deal. But we should clean it up some day, and handle empty selectors in this method.